INSTALL.md

```diff
@@ -107,3 +114,27 @@ This approach requires having [msys2](https://msys2.github.io/) installed.
   generate "projects" for your compiler.
 - Build with your compiler of choice. The generated build files should have the
   protobuf3 dependency baked in.
+
+
+###HP-UX
+
+- Export folowing variables:
+```
+DEP_FLAGS=” -I/usr/local/include ”
+DEP_LDFLAGS=” -L/usr/local/lib ”
+CFG_FLAGS=” -mlp64 -gdwarf-2 -D_INCLUDE_STDC__SOURCE_199901 -D_REENTRANT -D_INCLUDE_XOPEN_SOURCE_EXTENDED \
+          -D_INCLUDE_HPUX_SOURCE \
+          -D_INCLUDE_XOPEN_SOURCE_520 \
```

Comment:
Those flags is required to build in HP-UX (including -D) flags. It need to select approriate HP-UX API implementations. So it is-a must be in Makefile. Without it any additional API not included by default. If I move those flags in platform_flags.h, most files does not compile: 
```
[C]       Compiling src/core/lib/support/string.c
In file included from /usr/include/sys/signal.h:42:0,
                 from /usr/local/CC/gcc-4.8.5/lib/gcc/ia64-hp-hpux11.31/4.8.5/include-fixed/sys/pthread.h:50,
                 from /usr/include/pthread.h:16,
                 from include/grpc/impl/codegen/sync_posix.h:39,
                 from include/grpc/impl/codegen/sync.h:60,
                 from include/grpc/support/sync.h:38,
                 from include/grpc/slice.h:38,
                 from ./src/core/lib/support/string.h:39,
                 from src/core/lib/support/string.c:34:
/usr/include/sys/siginfo.h:115:2: error: unknown type name 'sigval_t'
  __SIGINFO_DATA
  ^
In file included from /usr/local/CC/gcc-4.8.5/lib/gcc/ia64-hp-hpux11.31/4.8.5/include-fixed/sys/newsig.h:87:0,
                 from /usr/include/sys/signal.h:43,
                 from /usr/local/CC/gcc-4.8.5/lib/gcc/ia64-hp-hpux11.31/4.8.5/include-fixed/sys/pthread.h:50,
                 from /usr/include/pthread.h:16,
                 from include/grpc/impl/codegen/sync_posix.h:39,
                 from include/grpc/impl/codegen/sync.h:60,
                 from include/grpc/support/sync.h:38,
                 from include/grpc/slice.h:38,
                 from ./src/core/lib/support/string.h:39,
                 from src/core/lib/support/string.c:34:
/usr/include/ia64/sys/newsig.h:104:2: error: unknown type name 'sigset_t'
  sigset_t __uc_sigmask; /* defines the set of signals that
  ^
In file included from /usr/include/pthread.h:16:0,
                 from include/grpc/impl/codegen/sync_posix.h:39,
                 from include/grpc/impl/codegen/sync.h:60,
                 from include/grpc/support/sync.h:38,
                 from include/grpc/slice.h:38,
                 from ./src/core/lib/support/string.h:39,
                 from src/core/lib/support/string.c:34:
/usr/local/CC/gcc-4.8.5/lib/gcc/ia64-hp-hpux11.31/4.8.5/include-fixed/sys/pthread.h:1106:2: error: unknown type name 'sigset_t'
  extern int pthread_sigmask(int, const sigset_t *, sigset_t *);
  ^
/usr/local/CC/gcc-4.8.5/lib/gcc/ia64-hp-hpux11.31/4.8.5/include-fixed/sys/pthread.h:1106:52: error: unknown type name 'sigset_t'
  extern int pthread_sigmask(int, const sigset_t *, sigset_t *);

```
However, those flags is required also by third party (e.g protobuf).
So I will try to move this stuff completely in Makefile.